src/go/rdctl/pkg/paths/paths.go

```diff
@@ -41,6 +41,9 @@ type Paths struct {
 	Snapshots string `json:"snapshots,omitempty"`
 	// Directory containing user-managed containerd-shims
 	ContainerdShims string `json:"containerdShims,omitempty"`
+	// Previous location of Electron session data (e.g. cookies) up to Rancher Desktop 1.16.
```

Comment:
This was actually user data, not just session data. (The difference is that it also contains the singleton lock on Linux/mac).

Reply:
Oops, I've renamed it to `OldUserData` now.